build.gradle

```diff
@@ -18,20 +18,23 @@ plugins {
     id 'io.gitlab.arturbosch.detekt' version '1.0.0.RC6-4'
     id 'com.diffplug.gradle.spotless' version '3.10.0'
     id 'net.ltgt.errorprone' version '0.0.13'
+    id "com.adarshr.test-logger" version "1.2.0"
```

Comment:
After looking into what this is, I'm now excited 😄

Reply:
I didn't configure it at all, but even out of the box I think it looks better.